lib/view_model/transaction_details_view_model.dart

```diff
@@ -26,8 +34,16 @@ abstract class TransactionDetailsViewModelBase with Store {
       {required this.transactionInfo,
       required this.transactionDescriptionBox,
       required this.wallet,
-      required this.settingsStore})
+      required this.settingsStore,
+      required this.sendViewModel})
       : items = [],
+        RBFListItems = [],
+        transactionPriority = bitcoin!.getBitcoinTransactionPriorityMedium(),
+        newFee = bitcoin!.getFeeAmountForPriority(
+            wallet,
+            bitcoin!.getBitcoinTransactionPriorityMedium(),
```

Comment:
same points.
try to build for Monero and fix accordingly

Reply:
fixed